vars/defaults.yml

```diff
@@ -13,6 +13,8 @@ main_user_shell: "/bin/bash"
 # encfs_password: (required)
 friendly_networks:
   - ""
+# letsencrypt_email: (required)
```

Comment:
Maybe s/required/recommended/? I think it would be great if it worked without setting the email since the ACME protocol explicitly supports this.